openid-connect-server/src/main/java/org/mitre/openid/connect/request/ConnectRequestParameters.java

```diff
@@ -18,31 +18,30 @@
 
 public interface ConnectRequestParameters {
 
-	public String CLIENT_ID = "client_id";
-	public String RESPONSE_TYPE = "response_type";
-	public String REDIRECT_URI = "redirect_uri";
-	public String STATE = "state";
-	public String DISPLAY = "display";
-	public String REQUEST = "request";
-	public String LOGIN_HINT = "login_hint";
-	public String MAX_AGE = "max_age";
-	public String CLAIMS = "claims";
-	public String SCOPE = "scope";
-	public String NONCE = "nonce";
-	public String PROMPT = "prompt";
+	String CLIENT_ID = "client_id";
+	String RESPONSE_TYPE = "response_type";
+	String REDIRECT_URI = "redirect_uri";
+	String STATE = "state";
+	String DISPLAY = "display";
+	String REQUEST = "request";
+	String LOGIN_HINT = "login_hint";
+	String MAX_AGE = "max_age";
+	String CLAIMS = "claims";
+	String SCOPE = "scope";
+	String NONCE = "nonce";
+	String PROMPT = "prompt";
 
 	// prompt values
-	public String PROMPT_LOGIN = "login";
-	public String PROMPT_NONE = "none";
-	public String PROMPT_CONSENT = "consent";
-	public String PROMPT_SEPARATOR = " ";
+	String PROMPT_LOGIN = "login";
+	String PROMPT_NONE = "none";
+	String PROMPT_CONSENT = "consent";
+	String PROMPT_SEPARATOR = " ";
 
 	// extensions
-	public String CSRF = "csrf";
-	public String APPROVED_SITE = "approved_site";
+	String APPROVED_SITE = "approved_site";
 
 	// responses
-	public String ERROR = "error";
-	public String LOGIN_REQUIRED = "login_required";
+	String ERROR = "error";
+	String LOGIN_REQUIRED = "login_required";
```

Comment:
The `public` modifiers here were removed by IntelliJ IDEA. According to the [JLS (§9.4)](https://docs.oracle.com/javase/specs/jls/se8/html/jls-9.html#jls-9.4),
> Every method declaration in the body of an interface is implicitly public (§6.6). It is permitted, but discouraged as a matter of style, to redundantly specify the public modifier for a method declaration in an interface.